src/Avalonia.Native/Avalonia.Native.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <IsPackable Condition="'$([MSBuild]::IsOSPlatform(OSX))' == 'True'">true</IsPackable>
```

Comment:
The check is here for a reason

Reply:
ok, but that not address the issue https://github.com/AvaloniaUI/Avalonia/issues/4148#issue-642261119